charts/enterprise-metrics/CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## 1.4.1
 
+* [CHANGE] The compactor data directory configuration has been corrected to `/data`.
+  > **Note:** The compactor is stateless and no data stored in the existing data directory needs to be moved in order to facilitate this upgrade.
+  > For more information, refer to the [Cortex Compactor documentation](https://cortexmetrics.io/docs/blocks-storage/compactor/).
```

Comment:
You updated the Chart version to 1.4.2 but the changelog is under 1.4.1

Reply:
Good catch, thanks